SS2-Project/Assets/Starstorm2/Components/Events/StormController.cs

```diff
@@ -21,15 +21,89 @@ public class StormController : NetworkBehaviour
         public static bool shouldShowObjective = false; // weather radio? config setting?
         private static Color etherealTextColor = new Color(.204f, .921f, .561f);
 
+
+        // OLD SHIT FUC!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+        /// <summary>
+        /// 
```

Comment:
bruh wtf is this

Reply:
Please remove unecesary whitelines. and summary